lvgl_tft/disp_driver.c

```diff
@@ -7,7 +7,7 @@
 #include "esp_lcd_backlight.h"
 #include "sdkconfig.h"
 
-void *disp_driver_init(void)
+void *disp_driver_init(lv_disp_drv_t *drv)
```

Comment:
I haven't found a place where this function is called with the new `disp_driver_init(lv_disp_drv_t *drv)` signature

Reply:
In my [test project](https://github.com/lvgl/lv_port_esp32/tree/feat/new_driver_test) I'm calling `st7789_init` and `st7789_flush` directly, planning to remove the disp_driver_init and display_driver_flush all together.

Reply:
>  planning to remove the disp_driver_init and display_driver_flush all together.

good to hear!